src/domain/event/eventForm/ValidationSchema.ts

```diff
@@ -35,9 +38,35 @@ const createValidationSchemaYup = (
       .typeError(VALIDATION_MESSAGE_KEYS.DATE)
       .required(VALIDATION_MESSAGE_KEYS.DATE_REQUIRED)
       .test(
-        'isTodayOrInTheFuture',
-        VALIDATION_MESSAGE_KEYS.DATE_TODAY_OR_LATER,
-        isTodayOrLater
+        'isInTheFuture',
+        VALIDATION_MESSAGE_KEYS.DATE_IN_THE_FUTURE,
+        isFuture
+      )
+      .when(
+        ['occurrenceDate', 'occurrenceStartsAt'],
+        (
+          occurenceDate: Date,
```

Comment:
Small typo with occurrurrence :) .....hah even I can't write it :D

Reply:
fixed, the typo was in many places